components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java

```diff
@@ -774,4 +773,23 @@ private static boolean isFormatCorrect(String regularExpression, String domainNa
         Matcher m2 = p2.matcher(domainName);
         return m2.matches();
     }
+
+    /**
+     * Triggers post tenant delete for TenantMgtListener.
+     *
+     * @param tenantId int Tenant id.
+     * @param tenantUuid String Tenant unique identifier.
+     * @param adminUserUuid String Tenant admin user unique identifier.
+     * @throws StratosException  If trigger failed.
+     */
+    private static void triggerPostTenantDelete(int tenantId, String tenantUuid, String adminUserUuid)
+            throws StratosException {
+
+        for (TenantMgtListener tenantMgtListener : TenantMgtServiceComponent
+                .getTenantMgtListeners()) {
```

Comment:
Formatting issue. This line can be move into the previous line

Reply:
Fixed in https://github.com/wso2/carbon-multitenancy/pull/218